Procfile

```diff
@@ -1,3 +1,4 @@
-web: npm start
-clock: npm run start-clock
-worker: npm run start-worker
+clock: IS_HEROKU=true npm run start-clock
+release: IS_HEROKU=true npm run on-release
+web: IS_HEROKU=true npm start
+worker: IS_HEROKU=true npm run start-worker
```

Comment:
I think "procfiles" are very specific to Heroku and if this application is not run on Heroku, they are just ignored. Considering this, wondering if we need the flag?